frontend/src/metabase/css/core/colors.module.css

```diff
@@ -27,17 +27,19 @@
   --mb-color-shadow: rgba(0, 0, 0, 0.13);
   --mb-color-border: #eeecec;
   --mb-color-filter: #7172ad;
-  --mb-color-focus: #cbe2f7;
+  --mb-color-focus: var(--mb-base-color-blue-20);
   --mb-color-warning: #f9cf48;
   --mb-color-text-primary: var(--mb-color-text-dark);
   --mb-color-text-secondary: var(--mb-color-text-medium);
   --mb-color-text-tertiary: var(--mb-color-text-light);
-  --mb-color-text-selected: var(--mb-base-color-brand-40);
+  --mb-color-text-selected: var(--mb-base-color-white);
+  --mb-color-text-hover: var(--mb-base-color-brand-40);
   --mb-color-text-brand: var(--mb-base-color-brand-40);
   --mb-color-text-white: var(--mb-base-color-white);
   --mb-color-background: var(--mb-base-color-white);
-  --mb-color-background-selected: var(--mb-base-color-brand-10);
-  --mb-color-background-disabled: var(--mb-color-bg-light);
+  --mb-color-background-selected: var(--mb-base-color-brand-40);
+  --mb-color-background-hover: var(--mb-base-color-brand-10);
+  --mb-color-background-disabled: var(--mb-base-color-gray-10);
```

Comment:
Modify the disabled color. Previously, we used both `bg-light` and `border` for the disabled state. This made it impossible to style, so I selected a color that doesn't look too far off from 2 colors.